pkg/agent/openflow/pipeline.go

```diff
@@ -883,25 +951,37 @@ func (c *client) conjunctionActionFlow(conjunctionID uint32, tableID binding.Tab
 		ofPriority = *priority
 	}
 	conjReg := IngressReg
-	if tableID == EgressRuleTable {
+	labelRange := binding.Range{0, 31}
+	if _, ok := egressTables[tableID]; ok {
 		conjReg = EgressReg
+		labelRange = binding.Range{32, 63}
 	}
 	return c.pipeline[tableID].BuildFlow(ofPriority).MatchProtocol(binding.ProtocolIP).
 		MatchConjID(conjunctionID).
 		MatchPriority(ofPriority).
 		Action().LoadRegRange(int(conjReg), conjunctionID, binding.Range{0, 31}). // Traceflow.
-		Action().GotoTable(nextTable).
+		Action().CT(true, nextTable, CtZone).
+		LoadToLabelRange(uint64(conjunctionID), &labelRange).
+		CTDone().
 		Cookie(c.cookieAllocator.Request(cookie.Policy).Raw()).
 		Done()
 }
 
-// conjunctionActionFlow generates the flow to drop traffic if policyRuleConjunction ID is matched.
+// conjunctionActionDropFlow generates the flow to mark the packet to be dropped if policyRuleConjunction ID is matched.
+// Any matched flow will be dropped in corresponding metric tables.
 func (c *client) conjunctionActionDropFlow(conjunctionID uint32, tableID binding.TableIDType, priority *uint16) binding.Flow {
 	ofPriority := *priority
+	metricTableID := IngressMetricTable
+	if _, ok := egressTables[tableID]; ok {
+		metricTableID = EgressMetricTable
+	}
+	// We do not drop the packet immediately but send the packet to the metric table to calculate the rule metric.
```

Comment:
I think: s/to calculate the rule metric/to update the rule metrics

Reply:
Done.